docs/source/features/cupti_counters_analysis.rst

```diff
@@ -0,0 +1,80 @@
+Counters Analysis
+=================
+.. warning::
+    This is an experimental feature in PyTorch and Holistic Trace Analysis.
+
+Motivation and context
+----------------------
+
+Performance counters measured on the GPU kernels can provide insights on how to speed up GPU kernels, conduct `roofline analysis`_ and other low level optimizations. Profiling tools like `NSight Compute <https://developer.nvidia.com/nsight-compute>`_ provide the ability achieve this locally but they do not work well on remote application, jobs running on a cluster etc.
```

Comment:
We should not be making statements about the lack of or the presence of features in other tools. Nsight compute can do several things which HTA cannot do. Since this is documentation of HTA features and not a comparative analysis we should refrain from making statements about other tools. 

Nsight compute CAN work on remote applications if the environment is setup appropriately.